src/components/ContentWrapper.jsx

```diff
@@ -411,7 +413,11 @@ const ContentWrapper = (props) => {
         </div>
 
       </Sider>
-      <Layout>
+      <Layout
+        style={{
+          'margin-left': '210px',
+        }}
```

Comment:
This causes the content to not stretch when the Sidebar is closed. Is it ok?

Reply:
obviously no, I'll have a look

Reply:
we have fixed this manually changing the left margin. It's a bit hacky but we've written good tests to make sure it doesn't break.